components/sst_importer/src/sst_importer.rs

```diff
@@ -342,11 +340,20 @@ impl SSTImporter {
             event_iter.next()?;
             let iter_key = event_iter.key().to_vec();
 
+            let ts = Key::from_encoded(iter_key.clone()).decode_ts()?;
```

Comment:
Maybe `Key::decode_ts_from(iter_key.as_slice())`?

```suggestion
            let ts = Key::decode_ts_from(&iter_key)?;
```